Documentation/operations/upgrade.rst

```diff
@@ -398,6 +398,8 @@ Helm Options
   been using ``securityContext.extraCapabilities`` you do not need to do anything.
   If you were leveraging ``securityContext.extraCapabilities``, you need to review
   ``securityContext.capabilities.cilium_agent``.
+* The ``securityContext`` for Hubble Relay now applies to the container, not the pod. To update the security context of the pod, use ``podSecurityContext``.
+* The ``securityContext`` for Hubble Relay now defaults to drop all capabilities and run as non-root user.
```

Comment:
I think you'll need a new section for v1.14 for those? This PR is not making it to v1.13 as far as I understand.

Reply:
Oh right, good point!

Reply:
Should I go ahead and remove the v1.13 section?

Reply:
I believe maintainers do it as part as the preparation steps for the next release? I don't remember. Might be worth checking if there was any commit touching that section after we branched off from 1.13 in any case

Reply:
```
$ git log --oneline $(git merge-base master v1.13).. -- Documentation/operations/upgrade.rst
51063421d8a2 metrics: Remove unused ARP ping metrics
6be650b98007 docs: Bump Linux minimum version to 4.19.57
dcc458b5f423 hubble: Update the reason label for hubble_drop_total metric
50ed9386696a Make cilium pprof listen address configurable
a0eeb41b19fd Update Documentation/operations/upgrade.rst
fb98a03314bc Add upgrade note for hubble_policy_verdicts_total now listing L7 flows
5b9188b86456 k8s/watchers: Adjust CNP handlers to set ProcessStartTime for PolicyAdd
fda366730e4d docs: Document internalTrafficPolicy-related changes in the upgrade guide
```

Some of these should likely be reported to the new section

Reply:
```
$ git log --oneline $(git merge-base master v1.13).. -- Documentation/operations/upgrade.rst
51063421d8a2 metrics: Remove unused ARP ping metrics
6be650b98007 docs: Bump Linux minimum version to 4.19.57
dcc458b5f423 hubble: Update the reason label for hubble_drop_total metric
50ed9386696a Make cilium pprof listen address configurable
a0eeb41b19fd Update Documentation/operations/upgrade.rst
fb98a03314bc Add upgrade note for hubble_policy_verdicts_total now listing L7 flows
5b9188b86456 k8s/watchers: Adjust CNP handlers to set ProcessStartTime for PolicyAdd
fda366730e4d docs: Document internalTrafficPolicy-related changes in the upgrade guide
```

Some of these may be worth reporting to the new section, if they haven't been backported to 1.13.

Reply:
I pushed a commit to remove the v1.13 upgrade notes section. I'll double check that upgrade notes for v1.14 weren't added to the v1.133 section.

Reply:
All right, all of the commits in your message above have been backported to v1.13 so I think we are good to go.